### PR TITLE
Related items features

### PR DESCRIPTION
**Pull request by Gledinburgh**
Added a stylesheet to repo. Refactored my components to respond to style sheet. Added an OutfitItemsList on app level.

### CODE REVIEW

**Comment by greensteph** [comment]
@Gledinburgh I pulled this down and it looks like there is an error on my side. I'll attach it below. @jonathanAgarcia @rick-o-H Can one of you pull this down and let me know if you see it also. Or we can all take a look when we meet this morning. 

Screen Shot 2020-11-19 at 5 25 36 AM

File: client/src/App.js [hunk 1]
```diff
@@ -30,6 +31,7 @@ class App extends React.Component {
         <NavBar />
         <ProductOverview selectedProduct={this.state.selectedProduct} styles={this.state.selectedPoductStyles} />
         <div><RelatedItemsComparisonList selectedProduct={this.state.selectedProduct} productChangeMethod={this.handleProductChange}/></div>
+        <div><OutfitItemList/></div>
```
Comment (rick-o-H) [inline client/src/App.js:34]: outfits!!!!👕

File: client/src/RealatedComparisonComponents/OutfitItemList.jsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
+import React, {useState} from 'react';
+import RelatedProductCard from './RelatedProductCard.jsx';
+import axios from 'axios';
+import {Container, Card, CardDeck} from 'react-bootstrap';
+import {PlusCircle} from 'react-bootstrap-icons';
```
Comment (rick-o-H) [inline client/src/RealatedComparisonComponents/OutfitItemList.jsx:5]: Plus circle, I like it!!!!

**Review by rick-o-H — APPROVED** [review]
Awesome work Glenn!!!!!

**Pull request merged**